### ISSUE TITLE
Improve usability of the alt text control in the Studio image editor

### ISSUE THREAD

**Issue by ali-hugo** [openedx/platform-roadmap#492]
### Type of Request

Fast Track Change (small, low-risk improvements)

### Feature Description

The current UX of the alt text field in the Studio image editor is confusing and could be improved for clarity and accessibility.

Currently, the checkbox labeled “This image is decorative (no alt text required)” appears below the alt text input field. This ordering implies that the user should enter alt text before deciding whether the image is decorative, which doesn’t reflect the actual intent of the control.

## Proposed changes

1. Reorder the checkbox and alt text field
   - Move the “decorative image” checkbox above the alt text input.
   - When the checkbox is selected, hide or disable the alt text field.
   - When unchecked, show the alt text field as normal.

1. Shorten and simplify the checkbox label
   - Current: “This image is decorative (no alt text required)”
   - Proposed text: "Decorative image (no alt text required)"

<img width="1847" height="781" alt="Image" src="https://github.com/user-attachments/assets/8d18cc67-bf26-4c96-81da-cc7488a1abc3" />

### Status

New

### Proposed By

OpenCraft

**Comment by ali-hugo** [comment]
@openedx/openedx-product-managers Please take a look at these suggestions for improving the usability of the alt text control in the the Studio image editor.

**Comment by cassiezamparini** [comment]
I like this suggestion 👍

**Comment by ayub02** [comment]
👍

**Comment by ali-hugo** [comment]
@xitij2000 This ticket has been approved by product.
